README.md

```diff
@@ -1,43 +1,72 @@
+[![Published on NPM](https://img.shields.io/npm/v/@polymer/iron-fit-behavior.svg)](https://www.npmjs.com/package/@polymer/iron-fit-behavior)
 [![Build status](https://travis-ci.org/PolymerElements/iron-fit-behavior.svg?branch=master)](https://travis-ci.org/PolymerElements/iron-fit-behavior)
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://webcomponents.org/element/@polymer/iron-fit-behavior)
 
-_[Demo and API docs](https://elements.polymer-project.org/elements/iron-fit-behavior)_
+## IronFitBehavior
 
+`IronFitBehavior` positions and fits an element in the bounds of another
+element and optionally centers it in the window or the other element.
 
-## Polymer.IronFitBehavior
+See: [Documentation](https://www.webcomponents.org/element/@polymer/iron-fit-behavior),
+ [Demo](https://www.webcomponents.org/element/@polymer/iron-fit-behavior/demo/demo/index.html).
 
-`Polymer.IronFitBehavior` fits an element in another element using `max-height` and `max-width`, and
-optionally centers it in the window or another element.
+## Usage
 
-The element will only be sized and/or positioned if it has not already been sized and/or positioned
-by CSS.
+### Installation
 
-| CSS properties | Action |
-| --- | --- |
-| `position` set | Element is not centered horizontally or vertically |
-| `top` or `bottom` set | Element is not vertically centered |
-| `left` or `right` set | Element is not horizontally centered |
-| `max-height` set | Element respects `max-height` |
-| `max-width` set | Element respects `max-width` |
+```
+npm install --save @polymer/iron-fit-behavior
+```
+
+### In a Polymer 3 element
 
-`Polymer.IronFitBehavior` can position an element into another element using
-`verticalAlign` and `horizontalAlign`. This will override the element's css position.
+```js
+import {PolymerElement} from '@polymer/polymer/polymer-element.js';
+import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
+import {html} from '@polymer/polymer/lib/utils/html-tag.js';
 
-```html
-  <div class="container">
-    <iron-fit-impl vertical-align="top" horizontal-align="auto">
-      Positioned into the container
-    </iron-fit-impl>
-  </div>
+import {IronFitBehavior} from '@polymer/iron-fit-behavior/iron-fit-behavior.js';
+
+class SimpleFit extends mixinBehaviors(IronFitBehavior, PolymerElement) {
+  static get template() {
+    return html`
+      <style>
+        :host {
+          background: white;
+          border: 2px solid gray;
+        }
+      </style>
+      This element can be positioned using the features of IronFitBehavior.
```

Comment:
You should show an example of this...that's the point of the example, right? Someone to copy paste this and get an idea of what this behaviour does.